components/Metadata/Metadata.tsx

```diff
@@ -0,0 +1,74 @@
+import Head from 'next/head';
+import { ImageTypes } from 'types/Base';
+
+type MetadataProps = {
+    title: string;
+    description: string;
+    image: ImageTypes;
+    slug: string;
+};
+
+export const Metadata = ({
```

Comment:
can we add default props to this component, I think we'd want to return an empty string instead of what it's current'y returning`"null"`.

Reply:
Added default props, ideally these should never be used as this component requires specific data - fallbacks aren't great here. This is why I haven't made any of the props optional.

Reply:
Looks good, nice thinking on adding text instead of an empty string. Just one issue for the image object raised here -> https://github.com/torchbox/careers/pull/53#discussion_r823814431